app/models/challenge/bylder.rb

```diff
@@ -45,6 +72,14 @@ def associated_user_for_tri_challenge(challenger)
           challenger.buck_id
         )
       end
+
+      def associated_user_for_open_spot_challenge(spot:, performance:)
+        (find_challenge(spot: spot, performance: performance)&.users || []).first
+      end
+
+      def find_challenge(spot:, performance:)
```

Comment:
This seems unnecessary since it is only doing a `find_by`. When reading the code that uses this, I'd expect it to be doing something more

Reply:
I agree. Probably went a little overboard with this